tests/suites/storage/StoragePage.ts

```diff
@@ -0,0 +1,34 @@
+import {Locator, Page} from '@playwright/test';
+
+import {storagePage} from '../../utils/constants';
+import {selectContentTable} from '../../utils/selectContentTable';
+import {PageModel} from '../../models/PageModel';
+
+type EntityType = 'Groups' | 'Nodes';
+type VisibleEntityType = 'Degraded' | 'Out of space' | 'All';
+
+export class StoragePage extends PageModel {
+    readonly table: Locator;
+    protected readonly entityTypeSelector: Locator;
+    protected readonly visibleEntityTypeSelector: Locator;
+
+    constructor(page: Page) {
+        super(page, storagePage);
+
+        this.table = selectContentTable(this.selector);
+
+        this.entityTypeSelector = this.selector
+            .locator('.global-storage__controls .yc-radio-button')
```

Comment:
you should add extra props for semantic locators like `aria-role` or at least `data-testid` because using css classes is bad practice